Public/Src/Engine/UnitTests/Scheduler.IntegrationTest/ChangeAffectedInputTests.cs

```diff
@@ -55,19 +60,16 @@ public void DirectAffectedFileInputTest()
 
             ResetPipGraphBuilder();
             SchedulePipBuilder(pipBuilderA);
-            pipBuilderB.SetEnvironmentVariable(StringId.Create(Context.StringTable, "[Sdk.BuildXL]qCodeCoverageEnumType"), "DynamicCodeCov");
-            pipBuilderB.SetChangeAffectedInputListWrittenFilePath(changeAffectedWrittenFile);
+            pipBuilderB.SetChangeAffectedInputListWrittenFile(changeAffectedWrittenFile);
             SchedulePipBuilder(pipBuilderB);
 
-
             var inputChangesFile = CreateOutputFileArtifact();
             File.WriteAllText(ArtifactToString(inputChangesFile), ArtifactToString(aInput));
             Configuration.Schedule.InputChanges = inputChangesFile.Path;
-            Environment.SetEnvironmentVariable("[Sdk.BuildXL]qCodeCoverageEnumType", "DynamicCodeCov");

```

Comment:
Why removing all the enviroment reset of [Sdk.BuildXL]qCodeCoverageEnumType? They are here to test the cache behavior.

Reply:
They are not needed. You removed the env var from the SDK. 

---
In reply to: [330721429](https://github.com/microsoft/BuildXL/pull/977#discussion_r330721429) [](ancestors = 330721429)